colossus/src/main/scala/colossus/protocols/http/Header.scala

```diff
@@ -411,10 +411,9 @@ object Connection {
 }
 
 object ContentType {
-
+  val HeaderKey       = "Content-Type"
```

Comment:
This value is already defined in `HttpHeaders`

Reply:
i saw [this comment](https://github.com/dkarivalis/colossus/blob/7c1bceda598665523806e4f39ecbac4582b66c91/colossus/src/main/scala/colossus/protocols/http/Header.scala#L286) and figured it may be cleaner to have two (one for lower case, another for mixed case)

Reply:
ah just read in https://tools.ietf.org/html/rfc2616#section-4.2 that `[Header] Field names are case-insensitive`. going to switch to the already defined variable ^_^